notebooks-auth/test_thredds.ipynb

```diff
@@ -0,0 +1,1019 @@
+{
```

Comment:
<p>Sorry must be late but I still don't see what's new here compared to previous cell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/PAVICS-e2e-workflow-tests/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>